### DISCUSSION TITLE
[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed

### DISCUSSION THREAD

**Discussion by MohammedAbuObaid** [django-recaptcha/django-recaptcha#265] [Q&A]
@adamchainz 
@AltusBarry 
@cblignaut 
@phala 
@gustavp 

Hey everyone,

I'm trying to play with this Django reCAPTCHA module. I installed the module and did the basic required config as given in the project homepage

The reCAPTCHA module is displaying correctly on the form, but it is generating an error when I submit the form:

**Exception:**
<urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)>

**Exception Locatoin:**
/usr/local/Cellar/python@3.8/3.8.6/Frameworks/Python.framework/Versions/3.8/lib/python3.8/urllib/request.py, line 1353, in do_open

Can anyone help, please?

**Comment by adamchainz** [comment]
I'm afraid I don't help maintain this library any more. It's kinda rude to tag a bunch of people in an issue, maintainers will already be notified if subscribed to the repo.

**Comment by MohammedAbuObaid** [comment]
> I'm afraid I don't help maintain this library any more. It's kinda rude to tag a bunch of people in an issue, maintainers will already be notified if subscribed to the repo.



Please excuse me for tagging the project maintainers. I'm new here and I didn't know how it works

Anyway, can you please help me with the right/current project maintainer?

Thanks

**Comment by Bishwas-py** [comment]
You can check this article dude: [Install reCaptcha in Django site
](https://blog.webmatrices.com/django-recaptcha-v3/)

**Comment by Stormheg** [comment]
For anyone reading this discussion: this is most likely an issue with your Python installation being misconfigured, not with django-recaptcha.

Since the discussion is quite old, I'm closing this as outdated.
